Template and VF Components/QueryComponent.component

```diff
@@ -0,0 +1,28 @@
+<apex:component controller="QueryComponentController">
+    <apex:attribute name="inputValue" description="Attribute on the component." type="String" required="required" assignTo="{!inputQuery}"/>
+    <apex:pageBlock title="Query Results" id="table">
```

Comment:
we usually try to use Labels instead of hard coded values

Reply:
i will consider in future aasignments.